Documentation/devel/stage1-implementors-guide.md

```diff
@@ -28,32 +28,31 @@ This allows users to employ different containment strategies on the same host ru
 Entrypoints
 -----------
 
-### `rkt run` => "coreos.com/rkt/stage1/run"
+### `rkt run` => `coreos.com/rkt/stage1/run`
 
-0. rkt prepares the pod's stage 1 and stage 2 images and Pod Manifest under "/var/lib/rkt/pods/$uuid", acquiring an exclusive advisory lock on the directory
-1. chdirs to "/var/lib/rkt/pods/$uuid"
-2. resolves the "coreos.com/rkt/stage1/run" entrypoint via Annotations found within "/var/lib/rkt/pods/$uuid/stage1/manifest"
-3. executes the resolved entrypoint relative to "/var/lib/rkt/pods/$uuid/stage1/rootfs"
+0. rkt prepares the pod's stage 1 and stage 2 images and Pod Manifest under `/var/lib/rkt/pods/$uuid`, acquiring an exclusive advisory lock on the directory
+1. chdirs to `/var/lib/rkt/pods/$uuid`
+2. resolves the `coreos.com/rkt/stage1/run` entrypoint via Annotations found within `/var/lib/rkt/pods/$uuid/stage1/manifest`
+3. executes the resolved entrypoint relative to `/var/lib/rkt/pods/$uuid/stage1/rootfs`
 
 It is the responsibility of this entrypoint to consume the Pod Manifest and execute the constituent apps in the appropriate environments as specified by the Pod Manifest.
 
-The environment variable "RKT_LOCK_FD" contains the file descriptor number of the open directory handle for "/var/lib/rkt/pods/$uuid".
+The environment variable `RKT_LOCK_FD` contains the file descriptor number of the open directory handle for `/var/lib/rkt/pods/$uuid`.
 It is necessary that stage 1 leave this file descriptor open and in its locked state for the duration of the `rkt run`.
 
-In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entrypoint is a static Go program found at "/init" within the stage 1 ACI rootfs.
+In the bundled rkt stage 1 which includes systemd-nspawn and systemd, the entrypoint is a static Go program found at `/init` within the stage 1 ACI rootfs.
 The majority of its execution entails generating a systemd-nspawn argument list and writing systemd unit files for the constituent apps before executing systemd-nspawn.
 Systemd-nspawn then boots the stage 1 systemd with the just-written unit files for launching the contained apps.
-The "/init" program is essentially a Pod Manifest to systemd-nspawn + systemd.service translator.
+The `/init` program is essentially a Pod Manifest to systemd-nspawn + systemd.service translator.
```

Comment:
Suggest: "The `/init` program's primary job is translating a pod manifest to systemd-nspawn systemd.services."